### PR TITLE
sysinfo: fix the unit of maxrss on FreeBSD

### PR DESCRIPTION
**Pull request by iblislin**
The value from `rusage.ru_maxrss` is in kilobytes.

### CODE REVIEW

File: base/sysinfo.jl [hunk 1]
```diff
@@ -193,6 +193,12 @@ function set_process_title(title::AbstractString)
     Base.uv_error("set_process_title", err)
 end
 
+"""
+    Sys.maxrss()
+
+Get the maximum resident set size utilized.
```
Comment (tkelman) [inline base/sysinfo.jl]: may as well say here in what units

Comment (iblis17) [inline base/sysinfo.jl]: 👌

File: base/sysinfo.jl [hunk 2]
```diff
@@ -193,6 +193,12 @@ function set_process_title(title::AbstractString)
     Base.uv_error("set_process_title", err)
 end
 
+"""
+    Sys.maxrss()
+
+Get the maximum resident set size utilized in bytes.
+See also getrusage(2).
```
Comment (musm) [inline base/sysinfo.jl]: This second sentence is confusing since `getrusage` is not a julia API.

Comment (iblis17) [inline base/sysinfo.jl]: 👌

**Pull request merged**